### DISCUSSION TITLE
Wouldn't it be better to move the noBlankTarget lint rule from lint/a11y/noBlankTarget to lint/security/noBlankTarget?

### DISCUSSION THREAD

**Discussion by siosio34** [biomejs/biome#3194] [Ideas]
The [noBlankTarget](https://biomejs.dev/linter/rules/no-blank-target/) option is currently in lint/a11y scope. However, after reading the explanation, I think it would be more correct to place it in the lint/security scope. Is there a reason why the lint rule is in the lint/a11y scope?

**Comment by ematipico** [comment]
That's a good suggestion. I think it makes sense
